ansible/fanout_connect.yml

```diff
@@ -7,12 +7,16 @@
         when:
             - dut is not defined
 
+      - name: Convert dut to a list
+        set_fact:
+          dut_list: "{{ dut.split(',') }}"
+
       - block:
         - set_fact:
             server: "{{ inventory_hostname|lower }}"
             server_port: "{{ external_port }}"
 
-        - debug: msg="Connect {{ server }}:{{ server_port }} to {{ dut }}"
+        - debug: msg="Connect {{ server }}:{{ server_port }} to {{ dut_list }}"
```

Comment:
You can use `testbed_facts['duts']` instead of dut_list

Reply:
updated, thx.